packages/hash/frontend/src/shared/layout/layout-with-sidebar/sidebar.tsx

```diff
@@ -59,7 +60,7 @@ export const PageSidebar: FunctionComponent = () => {
       <TopNavLink
         icon={faHome}
         title="Home"
-        href="/"
+        href={`/${activeWorkspaceAccountId}`}
```

Comment:
Currently the workspace homepage is hosted at `/<workpace-account-id`, not `/`. This is probably something we want to change, but for consistency I've reflected the current state here.